src/styles/main.scss

```diff
@@ -1,3 +1,9 @@
-body {
-  margin: 0;
-}
+@import "./utiles/variable.scss";
```

Comment:
The folder name 'utiles' should be renamed to 'utils' for better naming convention.